modules/planning/tasks/deciders/path_bounds_decider/path_bounds_decider.cc

```diff
@@ -148,10 +147,8 @@ Status PathBoundsDecider::Process(
       AERROR << msg;
       return Status(ErrorCode::PLANNING_ERROR, msg);
     }
-    if (!lanechange_path_bound.empty()) {
```

Comment:
Why removing this check?

Reply:
There is a lanechange_path_bound.empty() check on line 146 and returning with an Error status if empty. No need to do redundant check here. @panjiacheng

Reply:
Got it. thx.